### ISSUE TITLE
Season Posters

### ISSUE THREAD

**Issue by darthShadow** [RatingPosterDB/rpdb-folders#50]
Hi,

Would it be possible to add season posters too?

For TV shows, adding quality badges to the complete show poster is problematic since there is no guarantee that all the episodes across the seasons were released in the same quality or have the audio/video characteristics like Dolby Vision/HDR/TrueHD/Atmos etc. In contrast to that, episodes in the same season almost always have the same characteristics.

Almost all rating sites also have seasonal ratings too, in addition to the show ratings, which would be another nice addition for the season poster.

Thanks for considering.

**Comment by jaruba** [comment]
> since there is no guarantee that all the episodes across the seasons were released in the same quality

But there is also no guarantee that all episodes of a single season will have the same quality.

Otherwise, in general I agree and it has been requested once before, the complexity of this is large though. Season posters is the easiest part, season badges on season posters is doable too. Season ratings has high complexity and although some rating sites do offer them, they are in the minority compared to all rating sites that are supported, adding such logic will further complicate support for more rating sites in the future too, and it will also slow down the ratings updater significantly. The value of season ratings are debatable too, there will be a lot less votes, and such ratings will not be available at all for a new season of a series (as seasons / episodes cannot be rated prior to their release) while series ratings would be available. More so season ratings will be very inaccurate through the progression of a season, as every episode influences these ratings a lot, but does not influence the overall series rating too much.

**Comment by darthShadow** [comment]
> But there is also no guarantee that all episodes of a single season will have the same quality.

In most cases, the episodes **will** have the same quality across a single season. Like bluray disc releases or streaming releases. This is mostly in the context of completed seasons. In-progress seasons may vary from episode to episode so I am fine with using the series posters with generic badges for those.

Season ratings is lower priority for me compared to season badges & season posters. That said, I can live with ratings only for completed seasons, in case that makes it easier. In-progress seasons can have the same ratings as the series.

**Comment by jaruba** [comment]
I can't make any promises, am pretty strapped with time lately and this feature has high complexity. One thing I personally don't like is the complexity it brings to the entire project, making it harder to maintain and reducing sustainability.

I will leave the issue open, and I won't say no to the task yet, but you should not expect this any time soon.

**Comment by darthShadow** [comment]
Yeah, it may be complex but it will give me an excuse to increase my subscription amount (and probably for others too) since there are a lot more seasons than there are shows 😉 .

Another thing that could be incorporated into this idea is title cards like this (with just the season number and not the episode number): https://github.com/CollinHeist/TitleCardMaker#built-in-card-types

**Comment by darthShadow** [comment]
On further thought, just the series poster with the appropriate season name/number on the poster like the above title cards (https://github.com/CollinHeist/TitleCardMaker/wiki/Using-the-Mini-Maker#season-posters & https://github.com/CollinHeist/TitleCardMaker/wiki/Season-Posters) will be enough for now.

You could leave adding unique season posters & ratings for a later period when you have the time.

---

Not pushing for this, just a thought.

**Comment by darthShadow** [comment]
Hi, any further thoughts on this? Season Posters will be very helpful.

**Comment by darthShadow** [comment]
Bump.
